web/api/rest-api-design/Error-Response-Strategy.md

```diff
@@ -100,11 +153,20 @@ You can tie individual errors to the invalid request using a “parameters/path
 }
 ```
 
-### Internationalizing errors
+### Add a traceID to help trace errors
```

Comment:
Should this maybe be a part of headers so that it can be set in middleware and not have to do it in every handler. Though as you pointed out, it might be more visible to devs in the response body.

Reply:
I'd encourage us to use something other than "trace" as a descriptor here, as "tracing" is developing a larger meaning within our industry. My suggestion would be something akin to a "requestID".

And I agree with @felipe-lee that the `requestID` should be present in ALL responses, probably via header. 

(NB: I even screwed up naming the header in MilMove by using the `X-` prefix, which is now recommended against: https://datatracker.ietf.org/doc/html/rfc6648)